RPIPs/RPIP-36.md

```diff
@@ -0,0 +1,94 @@
+|   2023-04-09 | NonFungibleYokem | Voting      | [2023 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x19504a2bc3ac5b2b4fe9f54ef08ca23656fb4f6e678d17adb68c82d3bdf369af)                                                 | [Address](https://etherscan.io/address/0x435CA60b78df3E3d5A2042db60F90739680744Db) |
+|   2023-04-09 | phils.eth        | Voting      | [2023 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x19504a2bc3ac5b2b4fe9f54ef08ca23656fb4f6e678d17adb68c82d3bdf369af)                                                 | [Address](https://etherscan.io/address/0x163aDf9feCB5307762F8bbc90EDeF0054d5d04fB) |
+|   2022-08-25 | Jasper           | Voting      | [2023 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x19504a2bc3ac5b2b4fe9f54ef08ca23656fb4f6e678d17adb68c82d3bdf369af)                                                 | [Address](https://etherscan.io/address/0x04BEE613690e98A1959F236c38AbAa5f2439B14a) |
+|   2022-08-25 | Valdorff         | Voting      | [2023 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x19504a2bc3ac5b2b4fe9f54ef08ca23656fb4f6e678d17adb68c82d3bdf369af)                                                 | [Address](https://etherscan.io/address/0x68DEFBfFffE6e05e58A01eb2A12dEf7151C83bE2) |
+|   2022-08-25 | Uisce            | Voting      | [2023 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x19504a2bc3ac5b2b4fe9f54ef08ca23656fb4f6e678d17adb68c82d3bdf369af)                                                 | [Address](https://etherscan.io/address/0xB2b9bC63D475009cCc34956F1C733342bE60a6b6) |
+|   2022-08-25 | denum            | Voting      | [2023 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x19504a2bc3ac5b2b4fe9f54ef08ca23656fb4f6e678d17adb68c82d3bdf369af)                                                 | [Address](https://etherscan.io/address/0xC315e20930C61018B87785C7528FBe64f2C2f078) |
+
+### Historic Membership
+
+| Date Changed | Members                                                                                                                    | Comment                                                                                                                                                                                     |
+|--------------|----------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|   2023-09-22 | Jasper, Valdorff, knoshua, Uisce, ramana, denum, NonFungibleYokem, phils.eth, DrWorm                                       | DrWorm selected as [2023 IMC Full Selection](https://vote.rocketpool.net/#/proposal/0x19504a2bc3ac5b2b4fe9f54ef08ca23656fb4f6e678d17adb68c82d3bdf369af) runner-up after Langers steps down. |
```

Comment:
Maybe "Promoted as next backup from [Full Selection] after Langers stepped down"? I don't like "selected" which makes it sound like someone picked.